README.md

```diff
@@ -237,6 +237,56 @@ spec:
 
 Create a PVC using the storage class created for the ZFS driver.
 
+Here, we have to note that the size will be rounded off to the nearest Mi or Gi unit. Please note that M/G notation uses 1000 base and Mi/Gi notation uses 1024 base, so 1M will be 1000 * 1000 byte and 1Mi will be 1024 * 1024.
```

Comment:
We can probably push the detailed explanation into FAQ and just have a couple of lines about this in the main README.